frontend/src/features/permits/constants/tros.ts

```diff
@@ -9,11 +9,11 @@ import {
 
 export const TROS_INELIGIBLE_POWERUNITS = [...tros.ineligiblePowerUnitSubtypes];
 export const TROS_INELIGIBLE_TRAILERS = [...tros.ineligibleTrailerSubtypes];
-export const TROS_COMMODITIES: PermitCommodity[] = [...tros.commodities];
-export const MANDATORY_TROS_COMMODITIES: PermitCommodity[] = 
-  TROS_COMMODITIES.filter(
-    (commodity: PermitCommodity) =>
-      commodity.condition === "CVSE-1000" || commodity.condition === "CVSE-1070"
+export const TROS_CONDITIONS: PermitCondition[] = [...tros.conditions];
+export const MANDATORY_TROS_CONDITIONS: PermitCondition[] = 
+  TROS_CONDITIONS.filter(
+    (condition: PermitCondition) =>
+      condition.condition === "CVSE-1000" || condition.condition === "CVSE-1070"
```

Comment:
```suggestion
    ({ condition }: PermitCondition) =>
      condition === "CVSE-1000" || condition === "CVSE-1070"
```
I think this would make it easier to read?

Reply:
You're right, it is easier to read... I'll make the change